Makefile

```diff
@@ -1,4 +1,29 @@
+SHELL := bash
 .PHONY: clean
+
+.PHONY: install
+install:
+	poetry install
+
+.PHONY: test
+test:
+	poetry run pytest -vv --cov=dojo_toolkit --cov-report=term-missing
+
+.PHONY: lint
+lint:
+	@echo "Checking lock file"
+	poetry check --lock
+	@echo "Checking code"
+	poetry run ruff check .
```

Comment:
```suggestion
	poetry check --lock
	poetry run ruff check .
	poetry run ruff format . --check
```

Reply:
@zsinx6  I'm a bit confused:

Whats the difference between the commands:

```shell
ruff check
ruff format
```
?

Reply:
Found it:

https://github.com/astral-sh/ruff/discussions/9048

Thanks for the review!

Reply:
`ruff check` is for the violations set on `lint.select` here: https://github.com/grupy-sanca/dojo-toolkit/blob/3b0337c0a4d64712a05a329d1f8d8fe46b3bfb95/pyproject.toml#L34

`ruff format` is format and check the format (with `--check`), in summary it basically replaces `black`, you can read more details [here](https://docs.astral.sh/ruff/formatter/)